lib/asciidoctor/stylesheets.rb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
+# Monkey patch the whole file because the class is static
```

Comment:
Hmm. The plot thickens.

When I used the following, and compiled and ran it with Opal, it worked:

```ruby
require 'asciidoctor'

module Asciidoctor
class Stylesheets
  def primary_stylesheet_data
    @primary_stylesheet_data ||= 'foobar'
  end
end
end

puts Asciidoctor::Stylesheets.instance.primary_stylesheet_data
```

Note that with this approach, we need not worry about the path constants because we just have to return the data.